src/clojure/puppetlabs/ssl_utils/core.clj

```diff
@@ -583,8 +583,11 @@
    issuer-private-key :- PrivateKey
    issuer-public-key :- PublicKey
    cert-serials :- [schema/Int]]
-  (SSLUtils/revokeMultiple crl issuer-private-key
-                   issuer-public-key cert-serials))
+  (cond
```

Comment:
Since there are only two conditions you're checking, an `if` might make more sense.

Reply:
I might close this PR since this check can be perform over puppetserver repo as well. Thanks for the feedback!!

Reply:
I actually think it might make sense to keep this in this repo, since I feel like it's a more general improvement? But I could probably be convinced either way, depending on what it looks like in the puppetserver repo (haven't looked at your updates there yet 😅 ).

Reply:
ahh yeah, I guess this doesn't do the actual writing, so you'd still need the conditional in the puppetserver repo to not write unnecessarily (as Justin suggested), so yeah maybe it is best to just not make any updates in this repo.